hapi-fees/src/main/java/com/hedera/services/pricing/BaseOperationUsage.java

```diff
@@ -169,17 +177,25 @@ UsageAccumulator baseUsageFor(HederaFunctionality function, SubType type) {
 					return uniqueTokenBurn();
 				}
 				break;
-			case ConsensusSubmitMessage:
-				return submitMessage();
 			case TokenFeeScheduleUpdate:
 				return feeScheduleUpdate();
+			case ConsensusSubmitMessage:
+				return submitMessage();
 			default:
 				break;
 		}
 
 		throw new IllegalArgumentException("Canonical usage unknown");
 	}
 
+	UsageAccumulator fileAppend() {
+		/* The canonical usage and context */
+		final var opMeta = new FileAppendMeta(1_000, THREE_MONTHS_IN_SECONDS);
```

Comment:
The `FileAppendTransactionBody` doesn't have a field for (new) lifeTime. Does this imply that whenever a file has new appending contents, this file's expiry time will be automatically updated with the default value?

Reply:
Right, the "lifetime" of the new contents is exactly the current lifetime of the file as it exists!

Reply:
I see. If the user appends new contents to an existing file, the services will charge the user of the added contents storage bytes size with the default "lifetime" value.